docs/binary_format/example.hs

```diff
@@ -0,0 +1,96 @@
+module Example where
+
+import Data.Word
+import Data.ByteString.Lazy
+import Data.ByteString.Builder
+
+import Data.Void
+
+type Serialiser a = a -> Builder
```

Comment:
Are we consistent in spelling serilaiser vs. serializer in the codebase?

Reply:
We were, until I modified the JS api. I'll make a britishified version of the API too

Reply:
For some values of British. :9 🦁🇬🇧🦄